### PR TITLE
FLPATH-1526: link to troubleshooting guide + configmap update

### PR DESCRIPTION
**Pull request by jenniferubah**
Updated readme  file for MTA workflow
- Added link to the troubleshooting guide.
- Added info to update the `quarkus.rest-client.mta_json.url` field in the `mtaanalysis-props` ConfigMap
  Ran into issue where since the `mta.url` points to the external https route, the `quarkus.rest-client.mta_json.url` was still pointing 
  to ${mta.url} and it couldn't connect to it. It worked after I set `quarkus.rest-client.mta_json.url` to `http://mta-ui.openshift-mta.svc.cluster.local:8080/hub`.

### CODE REVIEW

File: docs/mta/README.md [hunk 1]
```diff
@@ -30,6 +31,16 @@ while [[ $retry_count -lt 5 ]]; do
 done
 echo "https://"$(oc -n openshift-mta get route mta -o yaml | yq -r .spec.host)
 ```
+Set the value of `quarkus.rest-client.mta_json.url` to `http://mta-ui.openshift-mta.svc.cluster.local:8080/hub`
+
+The mtaanalysis-props configmap should be similar to this:
+```console
+---
+mta.url = ${MTA_URL:https://<output-of-command-above>}
+quarkus.rest-client.mta_json.url = http://mta-ui.openshift-mta.svc.cluster.local:8080/hub
```
Comment (gabriel-farache) [inline docs/mta/README.md:40]: Where do you have that?
```
mta.url = ${MTA_URL:http://mta-ui.openshift-mta.svc.cluster.local:8080}
quarkus.rest-client.mta_json.url = ${mta.url}/hub
```
https://github.com/parodos-dev/serverless-workflows/blob/main/mta/application.properties

So, setting `mta.url` should be enough? Is it not?

Comment (gabriel-farache) [reply docs/mta/README.md:40]: also https://github.com/parodos-dev/serverless-workflows-config/blob/main/charts/mta-v7.x/INSTALL.md#edit-the-mta-analysis-v7-props-configmap

**Comment by jenniferubah** [comment]
Tested with older version of MTA workflow. Will re-create the PR after testing with the latest, MTAv7

**Pull request closed without merging**